### PR TITLE
Adding profile info

### PR DESCRIPTION
**Pull request by corey-taylor**
Adding my team text and photograph.

### CODE REVIEW

File: data/team/members.yaml [hunk 1]
```diff
@@ -75,6 +75,32 @@
...
+      url: https://github.com/corey-taylor
+    - icon: graduation-cap
+      text: Google Scholar
+      url: https://scholar.google.com/citations?user=IjR7hksAAAAJ&hl=en
+    - icon: book
+      text: ORCiD
+      url: https://orcid.org/0000-0003-2535-9514
+#    - icon: twitter
+#      text: Twitter
+#      url: https://twitter.com/jaime_rgp
```
Comment (dominiquesydow) [inline data/team/members.yaml:103]: I suggest to remove these commented lines (at least the jaime_rgp part).

**Review by dominiquesydow — APPROVED** [review]
Looks good to me; only change I suggest is removing the commented lines but that is up to you!

**Comment by dominiquesydow** [comment]
I think you should also be part of this team list, right?
https://volkamerlab.org/projects/kinoml/

**Comment by corey-taylor** [comment]
On the commented lines, yup I'm happy to remove them and add myself to the other page. How does the team do things? A new PR or one of the others ways (force push, squash, etc.)?

**Comment by dominiquesydow** [comment]
> A new PR or one of the others ways (force push, squash, etc.)?

In this case here, I would do this bit 
> yup I'm happy to remove them and add myself to the other page

in this PR (aka your `corey-taylor:master` branch) and then merge this PR into the `volkamerlab:master` branch (as you have defined for this PR).

---

Usually, we work on the `volkamerlab` repository directly (not on a fork of this repo) and would create PRs from `volkamerlab:ds-update-profile` to `volkamerlab:master`. Examples:
https://github.com/volkamerlab/volkamerlab_org/pull/57
https://github.com/volkamerlab/volkamerlab_org/pull/55

If this was not the question you were asking, please let me know :D

**Comment by corey-taylor** [comment]
Yup fair enough, I just followed the instructions where it said to fork to my Git then commit/push/PR. 

Can you provide some command liner examples just so I get it right next time?

**Comment by dominiquesydow** [comment]
Oh, so true! We do not follow our own README :D 

Here the procedure for this repo or a fork using branches (to not work on `master` directly):

```python
git clone https://github.com/volkamerlab/volkamerlab_org  # or use your fork
# create a new branch from the master branch (do your changes on that branch)
# give your branch your initials as prefix (convention that kind of stuck with us)
git checkout -b ct-profile
# do your changes
git add <your-changed-files>
git commit -m "Add Corey's profile"
git push origin ct-profile
# create your PR on GH (merging onto volkamerlab:master branch)
# get a review
# merge when you are ready
```

**Comment by corey-taylor** [comment]
Alright, cheers for that. Will merge this one for now but do it the other way next time.

**Pull request merged**